src/main/java/com/ldtteam/structures/helpers/Settings.java

```diff
@@ -321,6 +325,15 @@ public void moveTo(final BlockPos pos)
     @Nullable
     public Blueprint getActiveStructure()
     {
+        if (this.blueprint == null && this.structureName != null && !this.structureName.isEmpty())
+        {
+            final IStructureHandler structure = new CreativeStructureHandler(Minecraft.getInstance().level, new BlockPos(0, 0, 0), structureName,
+              new PlacementSettings(Settings.instance.getMirror(), BlockUtils.getRotation(Settings.instance.getRotation())), true);
+            if (structure.hasBluePrint())
+            {
+                this.blueprint = structure.getBluePrint();
+            }
+        }
```

Comment:
Kinda wondering if adding logic to Settings is gonna break something or not. If you checked the usage of this method, then mergeable

Reply:
Yes, I also tested it in smp+ssp